plugins/kubectl/kubectl.plugin.zsh

```diff
@@ -19,6 +19,9 @@ alias kca='_kca(){ kubectl "$@" --all-namespaces;  unset -f _kca; }; _kca'
 # Apply a YML file
 alias kaf='kubectl apply -f'
 
+# Apply a kustomization
+alias kak='kubectl apply -k'
```

Comment:
`kak` is the command for [`kakoune` editor](http://kakoune.org/), we'd need another name.